dsa/src/privatekey.rs

```diff
@@ -0,0 +1,200 @@
+//!
```

Comment:
Nit: we generally call these modules `private_key.rs` which follows a camel case -> snake case conversion